### ISSUE TITLE
Bring back theme.HyperlinkColor

### ISSUE THREAD

**Issue by dweymouth** [fyne-io/fyne#3867]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

Currently hyperlinks get their color from `theme.PrimaryColor`, but there used to be a `theme.HyperlinkColor`. I propose bringing back HyperlinkColor, to allow more customization while keeping in line with Fyne's theme API design philosophy.

In my app's case, since I have a lot of hyperlinks (most text in my app is links), I needed to darken the color of hyperlinks in the app's light theme to have appropriate text contrast. However, this makes the color of other PrimaryColor themed things, especially HighImportance buttons, a little jarringly dark.

### Describe the solution you'd like to see.

Bring back `theme.HyperlinkColor` as a standard color name in the theme API

**Comment by dweymouth** [comment]
<img width="229" alt="Screen Shot 2023-05-12 at 10 54 51 AM" src="https://github.com/fyne-io/fyne/assets/5303666/12090003-fcb8-4e98-9b90-91005e9c9efb">

Here's a screenshot from Supersonic in the light mode to better illustrate. The hyperlink text and the HighImportance button (used as a segmented control) are the same color. Note that the button looks darker, almost jarringly dark, compared to the text color which looks appropriate.

**Comment by andydotxyz** [comment]
This was deprecated back in 1.4.0 around 3 years ago.
We could indeed do as described above but I would be keen to get others thoughts on this topic...

**Comment by andydotxyz** [comment]
Curious about thoughts from @Jacalz @Bluebugs or @changkun

**Comment by Jacalz** [comment]
I don't really have anything against adding it back. The explanation above about text being smaller and needing different contrast makes sense to me.

**Comment by changkun** [comment]
I am actually interested in why hyperlink color not being customizable but have to define a separate const. Did I miss anything?

**Comment by andydotxyz** [comment]
With the old theme API adding a new item was a breaking change and when we moved to the new API we removed some for consistency.

Which means that now it is always primary and to add the customisation back in we need to introduce a new colour name that mirrors the old theme function. (Probably with the same helper at the package level but this time it won't be a breaking change to add it as v2 theme api is extensible).
